src/main/java/com/icthh/xm/uaa/service/AccountService.java

```diff
@@ -49,13 +59,13 @@ public class AccountService {
      */
     @Transactional
     @LogicExtensionPoint("Register")
-    public User register(ManagedUserVM user, String ipAddress) {
-        userService.validatePassword(user.getPassword());
-        String encryptedPassword = passwordEncoder.encode(user.getPassword());
+    public User register(UserDTO user, String ipAddress) {
```

Comment:
I worry about existing lep that can execute next code
ManagerUserVm user = lepContext.inArgs.user

Reply:
ManagerUserVm is an extension of UserDTO, so if ManagedUserVM is passed, not problem should occur